pino.js

```diff
@@ -36,6 +37,11 @@ var defaultOptions = {
   messageKey: 'msg'
 }
 
+var defaultBase = {
```

Comment:
Why isn't this part of the `defaultOptions` object?

Reply:
I simply renamed `defaultBase` according to the comments in #282 (from a previously submitted PR). I don't see a reason why the `base` object can't be a part of `defaultOptions`. Would you rather have something like:

Reply:
Yes, please.